release-notes.txt

```diff
@@ -1,3 +1,149 @@
+RIOT-2018.07 - Release Notes
+============================
+RIOT is a multi-threading operating system which enables soft real-time
+capabilities and comes with support for a range of devices that are typically
+found in the Internet of Things: 8-bit and 16-bit microcontrollers as well as
+light-weight 32-bit processors.
+
+RIOT is based on the following design principles: energy-efficiency, soft
+real-time capabilities, small memory footprint, modularity, and uniform API
+access, independent of the underlying hardware (with partial POSIX compliance).
+
+RIOT is developed by an international open-source community which is
+independent of specific vendors (e.g. similarly to the Linux community) and is
+licensed with a non-viral copyleft license (LGPLv2.1), which allows indirect
+business models around the free open-source software platform provided by RIOT.
+
+
+About this release:
+===================
+
+**DRAFT TODO**
+
+The 2018.07 release includes ...
+....
+....
+....
+
+> RMME: These are infos from 2018-07-12
+> RMME: Update them with: 'dist/tools/release-stats/release-stats.sh'
+
+About 320 pull requests with about 685 commits have been merged since the last
+release and about 54 issues have been solved. 44 people contributed with code
+in 85 days. Approximately 1648 files have been touched with 146204 insertions
+and 15683 deletions.
+
+**DRAFT TODO**
+
+
+Notations used below:
+=====================
+    + means new feature/item
+    * means modified feature/item
+    - means removed feature/item
+
+New features and changes
+========================
+
+Community
+---------
+    + Maintainers guidelines have been written and added to the repository
+
+Documentation
+-------------
+    + Fix undefined doxygen groups and add a dedicated static check
+    + Extend `auto_init` documentation
+
+Core
+----
+    + core: clist_foreach() now returns break-causing node
+
+System libraries
+----------------
+    +
+    -
+
+Networking
+----------
+    + asymcute: an asynchronous MQTT-SN client
+    + gnrc_ipv6: full duplicate address detection support according to
+      [RFC4862](https://tools.ietf.org/html/rfc4862)
+    + gnrc_ipv6: support to handle RDNSS option in neighbor discovery
+    + netif.h: a network-stack agnostic API to network interfaces
+    * sock_udp: allocate ephemeral on sock creation when port is 0
+    * **TODO* gcoap: List improvements
+    -
```

Comment:
Not sure but I think the API changes of the lorawan package was added in this cycle. Same for link_check feature

Reply:
`git log` agrees with you ;-):

```sh
$ git log --oneline -Slorawan 2018.07-devel..2018.07-branch
cf8ce68 tests/pkg_semtech_loramac: improve test documentation
c9f6013 examples: add a lorawan OTAA/class A example
5b76fdf pkg/semtech-loramac: refactor API to make it thread-safe
```